tests/utils/htmx/test.py

```diff
@@ -77,7 +77,7 @@ def update_page_with_htmx(page, select_htmx_element, htmx_response):
             _handle_swap(page, target=target, new_elements=[out_of_band_swap], mode=mode)
     _handle_swap(
         page,
-        target=_get_hx_attribute(htmx_element, "hx-target"),
```

Comment:
Le but c'est de ne pas mettre le hx-target quand c'est l'élément lui même, c'est ça ?

Reply:
C'est la logique d'`htmx` oui, par défaut le swap agira sur l'élément ayant déclenché la requête AJAX:

Cf https://htmx.org/attributes/hx-target/:

    The hx-target attribute allows you to target a different element for
    swapping than the one issuing the AJAX request.

Dans mon cas, c'est surtout que je fais un `hx-swap="none"` et donc que je n'ai pas défini de `hx-target`.
Et comme notre code ne gérait pas l'absence de `hx-target`, je rajoute son support en définissant la bonne valeur par défaut :sweat_smile: .